src/components/LoginRegister.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from 'react'
+import login_costume from '../assets/images/login-costume.svg'
+import { Link } from 'react-router-dom'
+import { Button, Form } from 'react-bootstrap'
+
+const URL = 'http://localhost:3001'
```

Comment:
change it to environment variable plss.
```suggestion
const URL = 'http://localhost:3001'
```